src/test/scala/uk/gov/nationalarchives/consignmentexport/BagAdditionalFilesSpec.scala

```diff
@@ -43,16 +43,18 @@ class BagAdditionalFilesSpec extends ExportSpec {
     val bagAdditionalFiles = BagAdditionalFiles(getClass.getResource(".").getPath.toPath)
     val lastModified: LocalDateTime = LocalDateTime.parse("2021-02-03T10:33:00.0")
     val fileMetadata = createMetadata(lastModified)
-    val metadata = createFile("originalPath", fileMetadata, "File", "name")
+    val metadata = createFile("originalPath", fileMetadata, "File", "name", "ref1")
     val file = bagAdditionalFiles.createFileMetadataCsv(List(metadata), customMetadata).unsafeRunSync()
 
     val source = Source.fromFile(file)
     val csvLines = source.getLines().toList
     val header = csvLines.head
     val rest = csvLines.tail
-    header should equal("File Path,File Name,File Type,File Size,Rights Copyright,Legal Status,Held By,Language,FOI Exemption Code,Last Modified Date,Checksum,OriginalFilepath")
+    println(header)
```

Comment:
Should these `println` statements be here?

Reply:
ah no it shouldn't